.gitignore

```diff
@@ -1,12 +1,10 @@
-!*.json
 *.pyc
 dist/
 build/
 connections.config
 sqlite.db*
-scripts/
 retriever.egg-info
 apidocs
 .DS_Store
 .cache/*
-*.bak
+scripts/*.py
```

Comment:
We should track changes to json scripts as often as we can, since they are not mature yet. For the python files that are compiled or written manually, we can force-add those files.